tests/saltify_test.py

```diff
@@ -74,4 +81,42 @@ def test_one(self):
         assert addConventions.addInflectionalMorphemes(x) == "He like/ed his[EP:her] shoe/s. \n"
     def test_two(self):
         x = "He looked at[EW] sad."
-        assert addConventions.addInflectionalMorphemes(x) == "He look/ed at[EW] sad. \n"
\ No newline at end of file
+        assert addConventions.addInflectionalMorphemes(x) == "He look/ed at[EW] sad. \n"
+
+class TestGrammarAndMorphemeFunctions(unittest.TestCase):
+
+    def setUp(self):
+        self.grammar_checker = grammar.GrammarChecker()
+
+    def read_file(self, file_path):
+        #read a file line by line
+        with open(file_path, 'r') as f:
+            return f.readlines()
+
+    def test_compare_input_with_output(self):
+        #load and read input and expected output files
+        input_file_path = os.path.join('tests', 'data', 'input1.txt')
```

Comment:
Create a function for handling multiple inputs/outputs. This will be helpful when working with multiple files.